src/commands/projects/create.ts

```diff
@@ -153,6 +153,10 @@ provide a description of your project using the --description flag.`
     const modes = asArray(options.modes)
     this.validateChannelsAndModes(channels, modes)
 
+    for (const channel of channels) {
+      validateChannelName(channel, '-' + (MixFlags.channelMultipleFlag.char?.toString() || 'flag'))
```

Comment:
Let's get rid of the "-". It actually looks like two hyphens that got auto-corrected to an en dash.